module/zfs/vdev_raidz_math_avx512f.c

```diff
@@ -241,6 +241,7 @@ typedef struct v {
 }
 
 #define	MUL_x2_DEFINE(x)						\
+/* cppcheck-suppress syntaxError */					\
```

Comment:
I think this may need to be,

```
// cppcheck-suppress syntaxError
```

Reply:
You're right. I was trying to figure out a way to get the supression comment to be included in the macro so `cppcheck` would actually suppress it and not break builds.

@behlendorf and I discussed offline and I'll have another attempted fix shortly.